packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWTag.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FC } from 'react';
+import { X } from '@phosphor-icons/react';
+
+import { CWCommunityAvatar } from '../cw_community_avatar';
+import { CWText } from '../cw_text';
+import { getClasses } from '../helpers';
+import { ComponentType } from '../types';
+import ChainInfo from 'client/scripts/models/ChainInfo';
+
+import 'components/component_kit/new_designs/CWTag.scss';
+
+type TagProps = {
+  community: ChainInfo;
+  disabled?: boolean;
+  onClick: () => void;
+};
+
+export const CWTag: FC<TagProps> = ({ community, disabled, onClick }) => {
+  const handleClick = () => {
+    onClick();
+  };
+
+  return (
+    <div className="CWTag">
```

Comment:
Ifu implemented tag already recently. You should use it instead of implementing your own

Reply:
I made some changes on Ifu's tag for the CWSearchBar to use it. Please let me know your thoughts.